apps/services/search-indexer/src/migrate/index.ts

```diff
@@ -9,6 +9,8 @@ interface PromiseStatus {
   error?: Error
 }
 
+export type locales = 'is' | 'en'
```

Comment:
Is this type not available from somewhere else already defined? and it would be better to upperfirst it